### PR TITLE
added todd motto and egghead tuts -> angularjs

### PR DESCRIPTION
**Pull request by aviaryan**

### CODE REVIEW

File: AngularJS/experts_to_follow.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name | Link
+------------ | ------------- 
+Todd Motto | https://twitter.com/toddmotto
```
Comment (PareshMayani) [inline AngularJS/experts_to_follow.md]: Also add other social media links (if available). For example, https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/experts_to_follow.md

**Comment by aviaryan** [comment]
Done

File: AngularJS/experts_to_follow.md (hunk 1 above)
Comment (aviaryan) [inline AngularJS/experts_to_follow.md]: Done

**Comment by PareshMayani** [comment]
Thank you @aviaryan for the contribution. Really appreciate it!

**Pull request merged**